x-pack/plugins/apm/public/components/app/trace_overview/trace_list.tsx

```diff
@@ -124,10 +125,11 @@ const noItemsMessage = (
   />
 );
 
-export function TraceList({ items = [], isLoading }: Props) {
+export function TraceList({ items = [], isLoading, errorFetching }: Props) {
   return (
     <ManagedTable
       isLoading={isLoading}
+      error={errorFetching}
```

Comment:
Another approach would be to pass in the status:
```suggestion
      status={status}
```